sirius-pipeline/bmv2/sirius_outbound.p4

```diff
@@ -32,7 +32,7 @@ control outbound(inout headers_t hdr,
     table routing {
         key = {
             meta.eni : exact @name("meta.eni:eni");
-            hdr.ipv4.dst_addr : lpm @name("hdr.ipv4.dst_addr:destination");
+            meta.dst_ip_addr : lpm @name("meta.dst_ip_addr:destination");
```

Comment:
There needs to be a flag to not overlap v4 and v6 prefixes that have identical 4 LSB.